dbt_subprojects/solana/models/_sector/dex/bot_trades/solana/platforms/alpha_dex_solana_bot_trades.sql

```diff
@@ -52,7 +52,7 @@ with
             tx_index,
             outer_instruction_index,
             inner_instruction_index
-        from {{ ref('dex_solana_trades') }} as trades
+        from {{ source('dex_solana', 'trades') }} as trades
```

Comment:
why are these defined as source?

Reply:
seems like you consistently and prob intentionally did that for the bot trades, but dex_solana_trades is defined in this project, won't that give an error?

Reply:
breaking up the lineage until we can get bot trades performance resolved.
not related to views being recreated, but a good time to implement this as i had been wanting to.
we don't rebuild bot trades on changes to dex anyway, so this removes the need for manual intervention.

Reply:
oh interesting that just breaks up the lineage dependency?

Reply:
today I learned, didn't know we can do that. 

in that case, all good